Frontend/src/components/TheNavbar.vue

```diff
@@ -154,7 +154,7 @@
             data-bs-toggle="dropdown"
           >
             <i class="bi bi-person"></i>
-            {{ loggedIn ? gettersAuthData.firstName : "Login" }}
+            {{ loggedIn ? authData.firstName : "Login" }}
```

Comment:
D.h. ich könnte jetzt "einfach" in Zeile 138 folgendes schreiben, um den Link zur Adminseite ein/auszuschalten:
`v-if=authData.role == "Moderator || authData.role == "Administrator`?

Evtl. wäre dann im Sinne der besseren Lesbarkeit noch ein Prop "hasElevatedRole" in authData cool, der prüft ob der User entweder Admin oder Mod ist.

Reply:
genau, das war der plan, aber in meinem branch war die admin Seite noch nicht drin, daher hatte ich das noch nichts drin.